lib/manageiq_performance/middlewares/memory.rb

```diff
@@ -0,0 +1,58 @@
+module ManageIQPerformance
+  module Middlewares
+    module Memory
+      private
+
+      def memory_initialize
+        require "objspace" if memory_include_memsize?
```

Comment:
Possibly.  Again, when posting, wasn't planning on getting an answer or even had a hypothesis for why, just making note of a observation.

I will say, I believe I made these measurements against a empty directory with only a gemfile that included the path to this project (not from within my `manageiq` bundle).  So anything loaded here would have been specifically from this gem.  Again, not a concern, just an observation.